public/index.html

```diff
@@ -22,6 +22,12 @@
       In current functionality we create history record if we pass a result field. It doesn't let make a comment on review. Because we decided to relate comments to history.
       -->
     <link rel="stylesheet" href="/styles/main.css">
+    <script>
+      // window.APP_SETTINGS = {
+      //   feature_flags: {},
+      //   feature_flags_default_value: true,
+      // }
+    </script>
```

Comment:
Is this needed at all, just saw this was commented out.

Reply:
We probably must enable it in the future for testing purposes. The LSE and LS work this way enabling all flags by default. This might help us catch errors earlier.

I disabled it for now as we don't have tests that cover all feature flags.